src/translations/en.json

```diff
@@ -2308,12 +2311,20 @@
                   "label": "Home Assistant",
                   "event": "Event:",
                   "start": "Start",
-                  "shutdown": "Shutdown"
+                  "shutdown": "Shutdown",
+                  "description": {
+                    "full": "When Home Assistant is {startedOrShutdown}",
+                    "started": "started",
+                    "shutdown": "shutdown"
```

Comment:
I'm not sure if we can do this. maybe the order of the sentence is completely different in some languages for started and shutdown for example.